airbyte-db/lib/src/main/java/io/airbyte/db/jdbc/DefaultJdbcDatabase.java

```diff
@@ -136,6 +136,10 @@ public void close() throws Exception {
 
   }
 
+  public CloseableConnectionSupplier getConnectionSupplier() {
```

Comment:
We have introduced this method to fetch the config. Instead we could refactor the code in `SnowflakeSqlOperations` to introduce a constructor which would receive the config as a parameter

Reply:
This is my mistake. This does not need to be changed, the problem is a different request. These changes are not relevant.